CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+- Fix version of `werkzeug`
```

Comment:
Can you update the PR title to make this more descriptive of the change for readers of the changelog?